packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -22,8 +22,7 @@ module.exports = {
                 '{Array,TypedArray}.prototype.toReversed',
                 '{Array,TypedArray}.prototype.toSorted',
                 '{Array,TypedArray}.prototype.with',
-                'Array.prototype.flat',
-                'Array.prototype.flatMap',
+                'Array.prototype.{flat,flatMap}',
```

Comment:
The merging of these is sensible but a breaking change

Reply:
This PR doesn't justify a major release on it's own, so hold for now.

Reply:
Backward compatibility would be possible based on assumption that if you've polyfilled one you've done the other. But adds complexity.